snooty/types.py

```diff
@@ -45,6 +45,9 @@ class SnootyError(Exception):
 
 @dataclass
 class StaticAsset:
+    # "key" must *exactly* match an identifier with which this asset is referred to in source text.
+    key: str
+
```

Comment:
[nit] Accidental newline?

Reply:
Intentional -- there's not really a good way to clearly document instance variables, so my convention is to have a newline after each documented instance variable